src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img"></div>
+
+      <p class="card__text card__text--head">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__text card__text--grey">Product code: 195434</p>
+
+      <div class="card__rating">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star stars__star--not_active"></div>
+        </div>
+
+        <p class="card__rating-text">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <h2 class="card__price-text">Price:</h2>
+        <p class="card__price-number">$2,199</p>
+      </div>
+      <a
```

Comment:
Anchor tags used as buttons should have a `role="button"` attribute for accessibility to indicate the role of the element.